doc/install/install-storage-cluster.rst

```diff
@@ -19,7 +19,9 @@ update APT's database and install Ceph::
 Installing with RPM
 ===================
 
-To install Ceph with RPMs, execute the following steps:
+To install Ceph with RPMs on RHEL 8 and derivatives the next steps are not needed because DNF support repository priorities natively.
+
```

Comment:
Run-on sentence is a bit awkward.  Suggest:

On RHEL 8 and derivatives the next steps are not needed because DNF natively supports repository priorities.